CMake/lrs_options.cmake

```diff
@@ -10,6 +10,7 @@ option(HWM_OVER_XU "Send HWM commands over UVC XU control" ON)
 option(COM_MULTITHREADED "Set OFF to initialize COM library with COINIT_APARTMENTTHREADED (Windows only)" ON)
 option(BUILD_SHARED_LIBS "Build shared library" ON)
 option(BUILD_UNIT_TESTS "Build realsense unit tests. Note that when enabled, additional tests data set will be downloaded from a web server and stored in a temp directory" OFF)
```

Comment:
`Build LibCI unit tests. If enabled, additional test data may be downloaded`

Reply:
done